new-lamassu-admin/src/pages/Authentication/Register.js

```diff
@@ -42,18 +42,17 @@ const REGISTER = gql`
   }
 `
 
-const validationSchema = Yup.object().shape({
+const validationSchema = Yup.object({
   password: Yup.string()
     .required('A password is required')
     .test(
       'len',
-      'Your password must contain more than 8 characters',
+      'Your password must contain at least 8 characters',
       val => val.length >= 8
     ),
```

Comment:
Suggestion: Instead using `test()` we could use `min()` method
```
.label("Password")
.min(8)
```

This will give *Password must be at least 8 characters* message.

For custom message we could write
```
 .min(8, "Your password must contain at least 8 characters")
```

Reply:
Neat, didn't know that! Will change it.